### DISCUSSION TITLE
Disable breakdowns on spiral slide

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18180] [Feature requests] [answered]
Breakdowns should be disabled on spiral slide because: 
1) there is no scenario where spiral slides are crucial to winning
2) the ride will not overperform with disabled breakdowns regarding money or stats
3) it is more realistic that it wouldn't break down since it is non mechanical and non electrical
4) If a spiral slide was broken, you wouldn't need a "mechanic" to fix it

**Comment by ZeeMaji** [comment]
I think it'd be an acceptable buff to an average flat ride, it wouldn't change a lot but it'd be a nice little buff for the spiral slide.

**Comment by 733737** [comment]
atm, I can't fix replays because when I click the load game button, my debug build crashes

**Comment by Basssiiie** [comment]
I think it'd be a pointless change. Why change it if it doesn't bother anyone? It's been part of the game for 20 years. It gives mechanics an extra task to do. It ties into the management part of the game.

Also I can think of various things that might break on a spiral slide due to a safety cut out:
- Lights inside the building are broken/off due to an electrical cut out.
- The indicator light for when you're allowed to go down the slide is broken/off.
- Some electric door, gate or maybe a [rotating security gate](https://v5i0d36g3mh1yj5rn3l51qnu-wpengine.netdna-ssl.com/wp-content/uploads/2017/04/LC100Black-300x300.jpg) is broken or shut down.
- Maybe there's an elevator for the slide mats that is broken down.

To be honest, I feel like some of your suggestions are a bit too much of wanting changes just for the sake of changing things, regardless of whether it has any good purpose. Not everything needs to be changed all the time. 😕

**Reply by 733737** [reply]
It depends what you want out of the game. Do you want a game that makes you feel like you're playing it in 2004, or do you want a game that feels fresh and exciting. Most people playing openrct2 are not new to rct, and have played it and similar games before

**Reply by Basssiiie** [reply]
Cuz removing the safety cut out from the spiral slide surely is refreshing and exciting.

**Reply by 733737** [reply]
just one piece in a puzzle

**Reply by Broxzier** [reply]
I agree with Basssiiie here. The inspections for spiral slide make sense. Just look at some pictures of defunct slides, clearly they would look better if they had some proper maintenanc. Inspections in-game for the spiral slide seem realistic enough to me.

**Reply by ZeeMaji** [reply]
I think i've changed my mind after reading basssiiie's potential safety cut out reason, makes sense.

**Comment by 733737** [comment]
I'm fine if the community disagrees, but have you ever looked at the sprites for the spiral slide? Maybe we should add all these electrical and mechanical things to the actual sprite, since the sprite has 1. no turnstiles 2. no operating booth 3. no device to lift mats to the top

**Reply by Broxzier** [reply, marked as answer]
I just imagine these are handled within the tower itself, or even at the entrance building.
